schema.sql

```diff
@@ -9,3 +9,17 @@ CREATE TABLE animals (
  weight_kg NUMERIC,
  species TEXT
 );
+
+
+CREATE TABLE owners(
+id SERIAL NOT NULL PRIMARY KEY,
+full_name VARCHAR,
+age INT);
+
+CREATE TABLE species(
+id SERIAL NOT NULL PRIMARY KEY,
+name VARCHAR);
```

Comment:
- [ ] KIndly add `PRIMARY KEY (id)` in tale creation.
```sql
CREATE TABLE species (
   id int NOT NULL IDENTITY(1,1),
   name varchar(255),
   PRIMARY KEY (id)
   );
```

Reply:
I already added the PRIMARY KEY on the same line as the id